src/routes/docs/configuring/ftp/+page.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Connecting to FTP server
+---
+
+Files supports connecting to remote FTP and FTPs servers.
+This allows you to effortlessly connect to any server that supports these protocols, browse the directory structure, and perform tasks such as uploading, downloading, renaming, and deleting files or folders according to your needs.
+
+_Currently Files only supports the FTP and FTPs protocols. SFTP is not supported yet._
```

Comment:
Maybe this should be put in a "Known limitations" section at the bottom of the page.